src/views/virtualmachines/actions/components/VMActionsIconBar/components/ActionIconButton.tsx

```diff
@@ -7,7 +7,13 @@ import { VMActionIconDetails } from '@virtualmachines/actions/components/VMActio
 
```

Comment:
data test id if very internal, what do u think about passing id? also you can pass id to the button and if needed create a data test id also

Reply:
Guohua asked for data-test-ids, so I think I'll pass id and use that for the data-test-id attribute.

Reply:
I updated it to use the VM action ids with '-button' as a suffix so I don't have to pass anything at all.